server/stats/game_stats_service.py

```diff
@@ -193,15 +193,12 @@ def _faction_played(self, faction, survived, achievements_queue, events_queue):
                 self._increment(ACH_SUTHANUS, 1, achievements_queue)
 
     def _killed_acus(self, unit_stats, survived, achievements_queue):
-        acus_per_player = unit_stats['cdr'].get('built', 1)
         killed_acus = unit_stats['cdr'].get('kills', 0)
 
-        # I'm aware that this is not perfectly correct, but it's edge case and tracking who got killed by whom would
-        # require game code changes
-        if killed_acus >= 3 * acus_per_player and survived:
-            self._unlock(ACH_HATTRICK, achievements_queue)
-
-        self._increment(ACH_DONT_MESS_WITH_ME, int(killed_acus / acus_per_player), achievements_queue)
+        if killed_acus > 0:
+            self._increment(ACH_DONT_MESS_WITH_ME, killed_acus, achievements_queue)
+            if killed_acus >= 3 and survived:
+                self._unlock(ACH_HATTRICK, achievements_queue)
```

Comment:
I think these two lines could be unindented one step

Reply:
My reasoning behind nesting them there was that if `killed_acus` is not greater than 0, the second `if` statement would not to be evaluated.

If you would still like them unindented one step, let me know and I'll update the MR.

Reply:
Sure, your choice :)

Reply:
> I'd be surprised if there was much of a difference in performance here.

Certainly nothing noticeable. :P